rollup/internal/controller/sender/sender.go

```diff
@@ -406,20 +413,35 @@ func (s *Sender) resubmitTransaction(feeData *FeeData, auth *bind.TransactOpts,
 		if gasFeeCap.Cmp(maxGasPrice) > 0 {
 			gasFeeCap = maxGasPrice
 		}
+
+		if originalGasTipCap.Cmp(gasTipCap) <= 0 {
```

Comment:
should it be `==`?

Reply:
changed in [4296d38](https://github.com/scroll-tech/scroll/pull/1058/commits/4296d38eb58699a7a9b397a68f543b3219de2b66)